src/algebraic_geometry/structure_sheaf.lean

```diff
@@ -799,6 +799,26 @@ def basic_open_iso (f : R) : (structure_sheaf R).presheaf.obj (op (basic_open f)
   CommRing.of (localization.away f) :=
 (as_iso (show CommRing.of _ ⟶ _, from to_basic_open R f)).symm
 
+instance is_iso_to_global (R : CommRing) : is_iso (structure_sheaf.to_open R ⊤ : R ⟶ _) :=
```

Comment:
```suggestion
instance is_iso_to_global : is_iso (structure_sheaf.to_open R ⊤) :=
```
The rest of this file works with an unbundled ring `R`, so I think it would be easier to keep it like that for now.